pkg/plugin/linuxutil/ethtool_stats_linux.go

```diff
@@ -67,10 +73,22 @@ func (er *EthtoolReader) readInterfaceStats() error {
 		if strings.Contains(i.Name, "lo") || strings.Contains(i.Name, "cbr0") {
 			continue
 		}
+
+		if er.unsupported[i.Name] {
+			er.l.Info("Skip Unsupported interface", zap.String("ifacename", i.Name))
+			continue
+		}
+
 		// Retrieve tx from eth0
 		ifaceStats, err := er.ethHandle.Stats(i.Name)
 		if err != nil {
 			er.l.Error("Error while getting ethtool:", zap.String("ifacename", i.Name), zap.Error(err))
+
+			if len(er.unsupported) < er.limit {
+				er.unsupported[i.Name] = true
+			} else {
+				er.l.Warn("Reached limit of unsupported interfaces")
+			}
 			continue
```

Comment:
What we're doing here is basically caching this error. I'd rather see a wrapper for `EthtoolInterface` that performs this caching, so that the handling is unchanged (it's still an error).